### DISCUSSION TITLE
Reading through the docs refences to public IP, support for only DNS/DDNS?

### DISCUSSION THREAD

**Discussion by FDrebin** [DefGuard/defguard#691] [General]
Reading through the docs before stand up and implementing in my homelab to replace an existing Wireguard server, the documents reference public IP a few times, how does the public IP impact deployment?

Can I access it exclusively via a DNS name I have a dynamic external IP?

**Comment by teon** [comment]
It should not be a problem - we have added in the last release the DNS name as endpoint for the server - see: https://github.com/DefGuard/wireguard-rs/commit/799775e4a76ee5f0c1f3af57fd9a65032ed23876
